src/views/NutritionView.vue

```diff
@@ -297,6 +302,39 @@ export default {
           ];
       }
     },
+    
+    getNutritionData: async function(){
+      const {lang, image_nutrition_url, images} = this.productBuffer[0];
+      let nutritionFromRobotService = await robotoffService.getNutritionValueFromImage(lang, image_nutrition_url, images);
+      this.nutritionData = nutritionFromRobotService.data.nutrients;
+      return this.nutritionData;      
+    },   
+
+    setNutritionData: function(){
+      const namingCorrection = {sugar: 'sugars', protein: 'proteins', carbohydrate: 'carbohydrates'};
+      const correctNaming = (robotoffNaming) => `nutriment_${namingCorrection[robotoffNaming] || robotoffNaming}`
+
+      for (const nutrimentId of Object.keys(this.nutritionData[0].data.nutrients)) {
+        let nutrimentKey = nutrimentId.replace(/_/g, '-');
+        let correctName = correctNaming(nutrimentKey);
+
+        if(nutrimentId=="energy"){
```

Comment:
good practice to allways use `===` instead of `==`
```suggestion
        if(nutrimentId === "energy"){
```